os/drivers/wireless/realtek/os/tizenrt/tizenrt_service.c

```diff
@@ -592,7 +592,16 @@ static int _tizenrt_create_task(struct task_struct *ptask, const char *name,
 								u32 stack_size, u32 priority, thread_func_t func, void *thctx)
 {
 	ptask->task_name = name;
-	ptask->task = kernel_thread(name, priority, stack_size, (main_t)func, NULL);
+	u32 priority_tmp = priority;
+	if (!strncmp(name, "recv_thread", sizeof("recv_thread"))) {
```

Comment:
Maybe we need to use 'strlen' instead of "sizeof", please check it.

Reply:
Why should we use `strlen`? `sizeof("recv_thread")` will be compiled as constant 10, and it is better than `strlen("recv_thread")`

Reply:
@junmin-kim  Hi, junmin. I didn't tell that we "should"...
I suggested him to check it to kernel team since both sizeof and strlen were found when I searched strncmp with grep in TizenRT.
And the result of sizeof("recv_thread") = 12 and that of strlen is 11..